engineering-feedback/feedback-faq.md

```diff
@@ -0,0 +1,39 @@
+# Engineering Feedback Frequently Asked Questions (F.A.Q.)
+
+The questions below are common questions related to the feedback process. The answers are intended to help both Microsoft employees and customers.
+
+## When should I submit feedback versus creating an issue on GitHub, UserVoice, or sending an email directly to a Microsoft employee?
+
+It is appropriate to do both. As a customer or Microsoft employee, you are empowered to create an issue or submit feedback via the medium appropriate for service.
+
+In addition to an issue on GitHub, feedback on UserVoice, or a personal email, Microsoft employees in CSE should submit feedback via CSE Feedback.  In doing so, please reference the GitHub issue, UserVoice feedback, or email by including a link to the item or attaching the email.
+
+Submitting to CSE Feedback allows the CSE Feedback team to coalesce feedback across a wide range of sources, and thus create a unified case to submit to the appropriate Azure engineering team(s).
+
+## How can a customer track the status of a specific feedback item?
+
+Customers are not able to track the status of feedback submitted via CSE Feedback.  The CSE Feedback process is internal to Microsoft, and as such, available only to Microsoft employees.
+
+However, as a customer, please submit your feedback via GitHub or UserVoice (as appropriate for the specific service), and inform your CSE engineering partner.  The CSE engineer should submit the feedback via the CSE Feedback process, and in doing so reference the previously created issue.  You can follow the GitHub or UserVoice item to be alerted on updates.
+
+## How can a Microsoft employee track the status of a specific feedback item?
+
+The easiest way for a Microsoft employee within CSE to track a specific feedback item is to [follow the feedback (a work item)](https://docs.microsoft.com/azure/devops/boards/work-items/follow-work-items?view=azure-devops) in Azure DevOps.
+
+## As a Microsoft employee within CSE, if I submit a feedback and move to another dev crew engagement, how would my customer get an update on that feedback?
+
+If the feedback is also submitted via GitHub or UserVoice, the customer may elect to follow that item for publicly available updates.
+
+## As a Microsoft employee within CSE, what should I expect/do after submitting feedback via CSE Feedback?
+
+After submitting the feedback, it is recommended to [follow the feedback (a work item)](https://docs.microsoft.com/azure/devops/boards/work-items/follow-work-items?view=azure-devops) in Azure DevOps.  If you have configured Azure DevOps notifications to send an email on work item updates, you will receive an email when the feedback is updated.
+
+If more information about the feedback is needed, a member of the CSE Feedback team will contact you to gather more information.
```

Comment:
Do we plan to go into what priorities there are and what each of they mean and how they are set?

Reply:
For priorities, maybe we add them to the guidance document?